paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -96,6 +97,9 @@
 
 AWS_SPOT_MODIFY_TIMEOUT = 30
 MISSING_SLAVE_PANIC_THRESHOLD = .3
+# Age threshold in seconds that should be met before an asg or sfr should
+# exceed before being checked for slave registration.
+CHECK_REGISTERED_SLAVE_THRESHOLD = 3600
```

Comment:
This should probably be defined as a getter on SystemPaastaConfig instead of a constant, so that when we want to tweak this we don't need to push a new version of Paasta.

Reply:
Good call.  I thought about this problem but didn't know where the best place to make it configurable was.

Reply:
My newest commit uses a new key in /etc/paasta/monitoring.json so we can deploy a tweak via puppet.  See https://reviewboard.yelpcorp.com/r/329219 and my latest commit.